tests/accordion-test.ts

```diff
@@ -0,0 +1,414 @@
+import { fixture, expect, html, oneEvent } from "@open-wc/testing";
+import "../src/accordion.ts";
+
+describe("AalamAccordion", () => {
+    it("initially opens items with the acc-active class", async () => {
+        const el = await fixture(html`
+            <aalam-accordion activecls="acc-active">
+                <div class="acc-title">Title 1</div>
+                <div class="acc-body acc-active">Content 1</div>
+                <div class="acc-title">Title 2</div>
+                <div class="acc-body">Content 2</div>
+            </aalam-accordion>
```

Comment:
If you could see the example in the specification, the children of aalam-accordion tag is represented wrongly. We need on child for every accordion item.